package.json

```diff
@@ -86,6 +86,11 @@
       "name": "Igor Savin",
       "email": "kibertoad@gmail.com",
       "url": "https://github.com/kibertoad"
+    },
+    {
+      "name": "Rafael Gonzaga",
+      "email": "rafael.nunu@hotmail.com",
+      "url": "https://github.com/rafaelgss"
     }
```

Comment:
This is out of order. The list is organized alphabetically by surname. This block should start on line 56.